media/css/mozorg/home/includes/mofo-donate-promo.scss

```diff
@@ -43,10 +43,6 @@ $image-path: '/media/protocol/img';
         font-weight: bold;
         text-align: center;
         margin: $spacing-lg 0;
-
-        span {
-            color: $color-violet-60;
-        }
```

Comment:
FYI I really haven't found any use of such h2 > span, so I'll double check if that was just some WIP stage during the PR or something that never made it to the final version…